clusters/l3-sqnc/secrets/github-dispatch.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+data:
+    token: ENC[AES256_GCM,data:VM6KVzijFh7p4hq5koh0ExQNaJLBPRiUtBPF1oFrR95IIHIPaSnCp2kurXl6ypncTirs2yIfnWhc9w7ulCAReoIdMqUelyYFmW2XADv7G4w5FgUV+5+UQs5ZCboKzyjQ06tkGC8gk871xI5IhmumrinC6WdESxwwlWXP2w==,iv:zr+x140/PvJSMSGAH3mQtum+3GbwFt4RRJDk+uYGFu8=,tag:5zqI5IFI97ISK4SrECLfXg==,type:str]
+kind: Secret
+metadata:
+    creationTimestamp: null
+    name: github-dispatch
+    namespace: monitoring
```

Comment:
Re-create this under the flux-system NS.

Reply:
done